pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1353,6 +1353,15 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 				if err != nil {
 					return err
 				}
+			} else if iface.Macvtap != nil {
+				if net.Multus == nil {
+					return fmt.Errorf("macvtap interface %s requires Multus meta-cni", iface.Name)
+				}
+
+				domainIface.Type = "ethernet"
+				if iface.BootOrder != nil {
+					domainIface.BootOrder = &BootOrder{Order: *iface.BootOrder}
+				}
```

Comment:
I see that the bridge/masquerade equivalent section sets the `Rom` to not enabled in case there is no boot order.
Why was it omitted here?

```
if iface.BootOrder != nil {
					domainIface.BootOrder = &BootOrder{Order: *iface.BootOrder}
				} else {
					domainIface.Rom = &Rom{Enabled: "no"}
				}
```

Reply:
That was introduced quite some time after this PR was opened; it intends to skip PXE rom on interfaces with no boot order, as per https://github.com/kubevirt/kubevirt/commit/d0187a6554290679822fb8f81ff448dc7f2b2ad9 . 

Will add it, since it seems I've not noticed this in one of the countless rebases done on this PR's life.